### DISCUSSION TITLE
Can this work for other 3d models displayd in Chrome, like Matterport 3D-scans from homes?

### DISCUSSION THREAD

**Discussion by Robonus** [eliemichel/MapsModelsImporter#225] [Ideas]
As seen on the Reddit page https://www.reddit.com/r/DataHoarder/comments/nycjj4/release_matterportdl_a_tool_for_archiving/
Matterport is a cloud-based 3d model (digital twin) from homes. Mostly used when you want to sell your house. That data cannot be downloaded or exported natively. Reddit user and "datahoarder" /u/rebane2001 has scripted a tool to copy the entire cloud environment to the local drive. (As for an example, see https://my.matterport.com/show/?m=JP9YgC9agCW for a big 3D scan model) But you still can't export the physical model to Blender or something similar. https://github.com/rebane2001/matterport-dl

A lot of us are looking for a way to download the model to an actual model, like *.OBJ or *.FBX. 
If your Google-maps downloader could be used for these models that would be awesome.
I've tried it on a way smaller model, and it looks like Renderdocs can grab the model and save it to an RDC file (about 90 MB, what seems about right)

However, the Blender plugin doesn't accept the RDC file from the Matterport model. Do you think it would be doable to fork the script so that it could import these models into Blender?

Kind regards!

**Comment by WingmanMedia** [comment]
Yeah, RenderDoc actually works and captures Matterport dollhouse mesh. I have just tried and had 0 issues. The problem is this Add-on for Blender 3D does not want to import it into Blender 3D because he looks at captures done with Google maps only. 

I am not sure if the developer is willing to make it work with any mesh in browser or at least with Matterport. However  if they do I would even pay for such extended version of this add-on something reasonable and there will be a lot of Matterport providers who would do the same.

I do not think anybody is going to use it just to get a mesh this way for free for a client. I personally would always pay and charge a client for a Matterpack when they need it.   However sometimes I want a mesh from my models for myself to play with in Blender3D and paying almost $100 for it is not an option.
